pkgs/development/tools/misc/coreboot-toolchain/default.nix

```diff
@@ -57,7 +58,9 @@ let
       buildPhase = ''
         export CROSSGCC_VERSION=$(cat .crossgcc_version)
         make crossgcc-${arch} CPUS=$NIX_BUILD_CORES DEST=$out
-      '';
+      '' + (lib.optionalString withIasl ''
+        make iasl CPUS=$NIX_BUILD_CORES DEST=$out
+      '');
```

Comment:
I think we should rework this part. setting `makeFlags` and so on instead of override buildPhase.

Reply:
Done. Moved the make flags into `makeFlags`.